ouroboros-consensus/src/Ouroboros/Consensus/BlockchainTime/WallClock/Types.hs

```diff
@@ -16,7 +16,6 @@ module Ouroboros.Consensus.BlockchainTime.WallClock.Types (
   , SystemTime (..)
     -- * Slot length
   , SlotLength
-    -- Opaque
```

Comment:
You can put this on the `-- * Slot length` line, like `-- * Slot length (opaque)`.